util/command_parser.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	"flag"
+	"fmt"
+)
+
+func ParseCommand(command []string) (map[string]string, error) {
+	fs := flag.NewFlagSet(command[0], flag.ExitOnError)
```

Comment:
I think you have to use flag.ContinueOnError otherwise you cannot get err when you call flag.Parse().
https://github.com/golang/go/blob/master/src/flag/flag.go#L940

Reply:
Thank you for notice. I've fixed to use ContinueOnError.